src/main/java/frc/robot/Robot.java

```diff
@@ -25,6 +27,13 @@
 
   private RobotContainer m_robotContainer;
 
+  //network table for vision control
+  private VisionCtrlNetTable m_VisionCtrlNetTable;
```

Comment:
Kind of a nitpick but to maintain consistency with the rest of the codebase these should be defined without underscores. For example `VisionControlNetTable visionControlNetTable`